### PR TITLE
LPS-191417 The default sort must be by the creation date

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
https://liferay.atlassian.net/browse/LPS-191417

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/odata/entity/v1_0/ObjectLayoutEntityModel.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+public class ObjectLayoutEntityModel implements EntityModel {
+
+	public ObjectLayoutEntityModel() {
+		_entityFieldsMap = EntityModel.toEntityFieldsMap(
+			new StringEntityField(
+				"name",
+				locale -> Field.getSortableFieldName(
+					"localized_name_".concat(
+						LocaleUtil.toLanguageId(locale)))));
+	}
```
Comment (mateussgn) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/odata/entity/v1_0/ObjectLayoutEntityModel.java:28]: The pattern here should not be `"label"` and `"localized_label_"`?

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/odata/entity/v1_0/ObjectViewEntityModel.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+public class ObjectViewEntityModel implements EntityModel {
+
+	public ObjectViewEntityModel() {
+		_entityFieldsMap = EntityModel.toEntityFieldsMap(
+			new StringEntityField(
+				"name",
+				locale -> Field.getSortableFieldName(
+					"localized_name_".concat(
+						LocaleUtil.toLanguageId(locale)))));
+	}
```
Comment (mateussgn) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/odata/entity/v1_0/ObjectViewEntityModel.java:28]: The pattern here should not be `"label"` and `"localized_label_"`?

**Review by mateussgn — APPROVED** [review]
Great Job, @pedro-oliveira446!!
LGTM! 🎉

**Comment by mateussgn** [comment]
Hey @MarinhoFeliphe  , could you please double-check this PR?

**Review by MarinhoFeliphe — APPROVED** [review]
LGTM!! :tada:

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Comment by pedro-oliveira446** [comment]
[Resent here](https://github.com/brianchandotcom/liferay-portal/pull/138519)

**Pull request closed without merging**